player/lua/osc.lua

```diff
@@ -157,7 +157,7 @@ function set_osd(res_x, res_y, text)
     state.osd.res_x = res_x
     state.osd.res_y = res_y
     state.osd.data = text
-    state.osd.z = 1000
+    state.osd.z = z or 1000
```

Comment:
Any reason to not just make the other `set_osd` call pass 1000 explicitly?

Reply:
I didn't want to touch all call sites, but I can change that if you'd prefer that.

Reply:
I personally think it's better to be explicit.

Reply:
I didn't realize there were only two call sites. Both of them are changed now.